lib/brcobranca/boleto/banco_brasil.rb

```diff
@@ -113,7 +113,7 @@ def numero_documento
       # @return [String] 1 caracteres numéricos.
       # @see BancoBrasil#numero_documento
       def nosso_numero_dv
-        "#{convenio}#{numero_documento}".modulo11_9to2_10_como_x
+        "#{convenio}#{numero_documento}".modulo11(mapeamento: {10 => 'X'})
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.